tasks.go

```diff
@@ -56,6 +62,34 @@ func GetAllTasks(basepath string) ([]Task, error) {
 	return tasks, nil
 }
 
+// GetNagiosTasks will return an array of Tasks each of which will dump the nagios data for one project.
+func GetNagiosTasks(projects []string, basepath string) ([]Task, error) {
+	var tasks []Task
+	var errors errorList
+	for _, p := range projects {
+		pods, err := getResourceNamesBySubstr(p, "pod", "nagios")
+		if err != nil {
+			errors = append(errors, err)
+			continue
+		}
+		for _, pod := range pods {
+			outFor := outToFile(basepath, "dat", "nagios")
+			errOutFor := outToFile(basepath, "stderr", "nagios")
+			task := GetNagiosStatusData(p, pod, outFor, errOutFor)
+			tasks = append(tasks, task)
+
+			outFor = outToFile(basepath, "tar", "nagios")
+			errOutFor = outToFile(basepath, "stderr", "nagios")
```

Comment:
yes, this is correct, the first set of tasks create:
```
nagios/<pod>-status.dat
nagios/<pod>-status.stderr
```
The second set of tasks create:
```
nagios/<pod>-history.dat
nagios/<pod>-history.stderr
```
But that is decided in the `getNagios(Status|History)Data` function in the nagios.go file.